packages/eslint-plugin/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@angular-eslint/eslint-plugin",
-  "version": "19.1.0",
```

Comment:
This shouldn't be here at all, there should not be any changes to package.json files in this PR